libs/transloco-schematics/src/join/schema.ts

```diff
@@ -25,5 +25,5 @@ export interface SchemaOptions extends Schema {
   /**
    * The root project name.
    */
-  project?: string;
+  project: string;
```

Comment:
Why did you make this change? 🤔

Reply:
New NX version is stricter. Getting this error when I leave `project` as optional:

![image](https://user-images.githubusercontent.com/655241/234362402-6a9737e9-4d9b-48ca-ac2e-accb8699b25a.png)

Reply:
`nx run transloco-schematics:build` command failed because of this inconsistency.